ui/src/views/user/login/LoginView.vue

```diff
@@ -121,16 +121,15 @@ export default {
       const userData = {
         accountType: 0,
         email: "user@email.com",
-        hasAcceptedPrivacyPolicy: false,
-        hasAcceptedTnC: false
       };
+      // this.$cookies.set("access_token_cookie", "test");  // can be set if needed, token is already set in vuex tho
       this.$store.dispatch("user/setToken", "test");
       this.$store.dispatch("user/setUserData", userData);
       this.$store.dispatch("user/checkAccountTypeAndSetMenuItems");
```

Comment:
Also one more question, here in `this.$store.dispatch` we have `user/setToken`, but in ui/src/store/modules/user.js I don't see any method called setToken, so I am not sure if it is working.

Reply:
Okay, I have been reading the code and I understand now that setToken is 'automatically' available by default for setting things in `state`, and you set `setUserData` manually to override the default setter because userData is not a regular variable, but a map. I mark this as resolved.

Reply:
yes, the methods are made automatically by default with the help of the make helper XD  no worries